src/ring_theory/jacobson.lean

```diff
@@ -10,70 +10,87 @@ import ring_theory.localization
 
 /-!
 # Jacobson Rings
-
 The following conditions are equivalent for a ring `R`:
-1. Every radical ideal `I` is equal to its jacobson radical
+1. Every radical ideal `I` is equal to its Jacobson radical
 2. Every radical ideal `I` can be written as an intersection of maximal ideals
-3. Every prime ideal `I` is equal to its jacobson radical
-
+3. Every prime ideal `I` is equal to its Jacobson radical
 Any ring satisfying any of these equivalent conditions is said to be Jacobson.
-
 Some particular examples of Jacobson rings are also proven.
 `is_jacobson_quotient` says that the quotient of a Jacobson ring is Jacobson.
 `is_jacobson_localization` says the localization of a Jacobson ring to a single element is Jacobson.
 `is_jacobson_polynomial_iff_is_jacobson` says polynomials over a Jacobson ring form a Jacobson ring.
-
 ## Main definitions
-
 Let `R` be a commutative ring. Jacobson Rings are defined using the first of the above conditions
-
 * `is_jacobson R` is the proposition that `R` is a Jacobson ring. It is a class,
   implemented as the predicate that for any ideal, `I.radical = I` implies `I.jacobson = I`.
-
 ## Main statements
-
 * `is_jacobson_iff_prime_eq` is the equivalence between conditions 1 and 3 above.
-
 * `is_jacobson_iff_Inf_maximal` is the equivalence between conditions 1 and 2 above.
-
 * `is_jacobson_of_surjective` says that if `R` is a Jacobson ring and `f : R →+* S` is surjective,
   then `S` is also a Jacobson ring
-
-* `is_jacobson_mv_polynomial` says that multi-variate polynomials over a jacobson ring are jacobson
-
+* `is_jacobson_mv_polynomial` says that multi-variate polynomials over a Jacobson ring are Jacobson.
 ## Tags
-
 Jacobson, Jacobson Ring
+-/
+
+namespace ideal
+
+open polynomial
 
+section technical_lemmas
+/-!
+The following two lemmas are independent of Jacobson.  I extracted them from the
+original proof of `quotient_mk_comp_C_is_integral_of_jacobson` to speed up processing.
+Honestly, I did not spend time to unwind their "meaning", hence the names are likely inappropriate.
```

Comment:
Thanks for doing this! We can leave them here for now, but in a future PR they should move to a better place.
```suggestion
Honestly, I did not spend time to unwind their "meaning", hence the names are likely inappropriate.

TODO: clean up, move elsewhere
```

Reply:
Sounds great, thanks!